src/@next/Pagination/PaginationStyle.tsx

```diff
@@ -58,6 +63,30 @@ export const StyledSimplePaginationButton = styled(StyledPageButton)`
   pointer-events: none;
 `;
 
+export const StyledBorderPaginationButton = styled(StyledPageButton)`
+  border: 1px solid ${Neutral.B68};
+
+  &:first-of-type {
+    border-radius: ${borderRadius2} 0 0 ${borderRadius2};
+  }
+
+  &:last-of-type {
```

Comment:
please add either `border-right-style: none` to the previous button OR `border-left-style:none` to the next button

Reply:
just tried this. this works out if there is no label given. but, if a label is given, it's obvious that either the right-side of prev button or the left-side of next button will miss a border. the result will be like this image below
<img width="153" alt="image" src="https://github.com/glints-dev/glints-aries/assets/3447145/6cabf1ba-d90b-4e32-a3a0-488c4768956b">

Reply:
Can we disable the "border-right-style" or "border-left-style" effect when label is given? @davidzzz

Reply:
btw @davidzzz we should not apply both "border-right-style" and "border-left-style" to the buttons. My suggestion was to pick one instead.

Reply:
@davidzzz wait! I found a way to achieve the desired outcome with css. give me a sec. let me test that out for the one with label first

Reply:
I will check more on this whether we can disable the "border-right-style" or "border-left-style" effect when label is given. will be a bit trickier to pull it off